### PR TITLE
106 log post server

### LINKED ISSUE #106: Get the log posting to the server
When user tap/click on the +add log feature a modal will appear and they can write their mood for particular activity. From the client side, we need create an action and a reducer to store that data in redux store.

Related #41 

closes #106

### PR DESCRIPTION
**Pull request by chrislaupama**
In this PR:

Changed the way the modal functions as well as turning the textarea into a proper form.

I also changed the submit into a form submit button, added change handlers and submitting the result of what's being entered as an object from superagent in Redux for the backend to use to save the data in the DB.

The "Add Log" currently works but we're just waiting on the backend to receive and updates these changes in the DB which I believe Dani will get working.

So in short, I think it's working but needs the backend to connect it up to the DB.

semi closes #106

### CODE REVIEW

**Review by CarelS072 — APPROVED** [review]
Looking good Chris!
